xapian-bindings/python3/Makefile.am

```diff
@@ -39,8 +39,7 @@ EXTRA_DIST = python.i util.i extra.i extracomments.i except.i \
 	run-python-test \
 	sphinx-build \
 	testsuite.py \
-	test_xapian_star.py \
-	$(TESTS) $(BUILT_SOURCES) $(RST_DOCS)
```

Comment:
I don't see any need to move these from here - make assignments aren't processed in file order (unless you use `:=` or similar) so it doesn't matter that the values of these are appended to below.

Reply:
It's because `BUILT_SOURCES` and `TESTS` are not finalized yet, otherwise `EXTRA_DIST` would include repeated files.

Reply:
fixed in a fixup commit.